cli/src/commands/bootstrap_config.py

```diff
@@ -19,9 +19,13 @@ def run(self, args):
 
         config_folder = environ.get('PUBLIC_CONFIG_FOLDER')
 
+        if config_folder is None:
+            print('PUBLIC_CONFIG_FOLDER must be defined in environment')
+
         if config_folder is None:
             self.print_config(config)
         else:
+            config_folder += '/configs'
```

Comment:
You should use https://docs.python.org/3/library/os.path.html for path management (`os.path.join`).